packages/ndla-accordion/src/Accordions.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useState, Children } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+// @ts-ignore
+import Button from '@ndla/button';
+
+import { getPanelIds, getOpenPanels } from './accordionUtil';
+import AccordionSection from './AccordionSection';
+
+type Props = { tiny?: boolean; single?: boolean };
+
+const Accordions: React.FC<Props & tType> = ({ tiny, single, t, children }) => {
+  const [panelIds] = useState<string[]>(getPanelIds(children));
+  const [openPanels, setOpenPanels] = useState<string[]>(getOpenPanels(children));
+
+  const toggleOpen = (id: string) => {
+    if (single) {
+      const newOpenPanels = openPanels.includes(id) ? [] : [id];
+      setOpenPanels(newOpenPanels);
+    } else {
+      const newOpenPanels = openPanels.includes(id)
+        ? openPanels.filter(p => p !== id)
+        : openPanels.concat(id);
+      setOpenPanels(newOpenPanels);
+    }
+  };
+  const allOpen = openPanels.length === panelIds.length;
+  const toggleAllOpen = () => {
+    const newOpenPanels = allOpen ? [] : [...panelIds];
+    setOpenPanels(newOpenPanels);
+  };
+
+  return (
+    <>
+      {!single && (
```

Comment:
Her viser du knappen kun dersom single === true. Burde det være mogeleg å velge om knappen skal vises? Eller antar vi at knapp for å åpne og lukke alle alltid skal være tilgjengelig?

Reply:
Det er ingen vits i å bruke "Accordions" dersom du ikke skal ha enten "single" eller "åpne/lukke" alle
Da kan du bare skrive AccordionSections alene. Accordions tar bare å tar seg av kommunikasjon mellom seksjonene når det er nødvendig

Reply:
Du har overbevist meg.